issue-wanted.cabal

```diff
@@ -111,6 +111,7 @@ library
                      , http-client-tls ^>= 0.3.5.3
                      , http-types ^>= 0.12
                      , lens ^>= 4.17
+                     , lifted-async ^>= 0.10.0.4
```

Comment:
Is this package needed? I see that we're using `MonadUnliftIO`, not `MonadBaseControl`